src/internal/NeoEspBitBangMethod.h

```diff
@@ -261,7 +261,8 @@ template<typename T_SPEED, typename T_PINSET> class NeoEspBitBangMethodBase
 public:
     NeoEspBitBangMethodBase(uint8_t pin, uint16_t pixelCount, size_t elementSize, size_t settingsSize) :
         _sizeData(pixelCount * elementSize + settingsSize),
-        _pin(pin)
+        _pin(pin),
+        _endTime(0)
```

Comment:
Is this just for satisfying a precheck script?  I can say that init it to zero is not a value that will be helpful at this point, random would be just as helpful.
The _endTime is correctly initialized in the the "initialize method".  BUT, there is a bug there I just noticed for the esp32 initialize for this issue.
Run a compile before this change and after this change, did it change the code size?  Memory size?

Reply:
I ran tests, and this change takes more program memory at no real bug fix.  I already fixed the one spot where it wasn't initialized.